src/components/DialoguesWithoutCuratorButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { ChatBubble } from '@mui/icons-material';
+import { Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+export const DialoguesWithoutCuratorButton = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const onClick = () => {
+    const statusNew: TicketStatus = 'New';
+    searchParams.set(TicketSearchParam.Status, statusNew);
+    searchParams.delete(TicketSearchParam.Assignee);
+    history.push({ search: searchParams.toString() });
+  };
+
+  return (
+    <Button
+      fullWidth
+      size="large"
+      variant="outlined"
+      color="primary"
+      sx={{ textTransform: 'none' }}
+      startIcon={<ChatBubble />}
+      onClick={() => onClick()}
+      title={t('tasks.dialogues_without_curator')}
+      data-testid={dataTestId.tasksPage.dialoguesWithoutCuratorButton}>
+      <Typography noWrap>{t('tasks.dialogues_without_curator')}</Typography>
+    </Button>
+  );
+};
```

Comment:
Hadde kanskje forventet at dette var en toggle? 
```suggestion
export const DialoguesWithoutCuratorButton = () => {
  const { t } = useTranslation();
  const history = useHistory();
  const searchParams = new URLSearchParams(history.location.search);
  const showOnlyUnassigned =
    (searchParams.get(TicketSearchParam.Status) === 'New' && searchParams.get(TicketSearchParam.Assignee) === null) ||
    undefined;

  const onClick = () => {
    const statusNew: TicketStatus = 'New';
    if (!showOnlyUnassigned) {
    searchParams.set(TicketSearchParam.Status, statusNew);
    searchParams.delete(TicketSearchParam.Assignee);
    } else { 
    searchParams.delete(TicketSearchParam.Status);
    }
    history.push({ search: searchParams.toString() });
  };

  return (
    <Button
      fullWidth
      size="large"
      variant={showOnlyUnassigned ? 'contained' : 'outlined'}
      color="primary"
      sx={{ textTransform: 'none' }}
      startIcon={<ChatBubble />}
      onClick={() => onClick()}
      title={t('tasks.dialogues_without_curator')}
      data-testid={dataTestId.tasksPage.dialoguesWithoutCuratorButton}>
      <Typography noWrap sx={{ color: showOnlyUnassigned ? 'white' : 'primary,main' }}>
        {t('tasks.dialogues_without_curator')}
      </Typography>
    </Button>
  );
};
```

Reply:
Enig i at dette muligens burde vært en toggle hvor man kan hoppe fram og tilbake, men det må bli ved en senere anledning etter en runde med designer og PO. Mtp deadline så sjekker jeg inn dette som er bestilt.

Reply:
Enig med Isar i at når jeg trykker på så ser det ikke ut som det egentlig har noen effekt, siden man ikke får noen respons på at den er valgt 🤔 
Kan den ikke vises som valgt (`contained`) når man søker etter status=new? For det er vel i praksis det denne gjør

Reply:
Burde faktisk holde med `variant={searchParams.get(TicketSearchParam.Status) === 'New' ? 'contained' : 'outlined'}` tror jeg

Reply:
Okei, to mot èn, jeg gir meg 😄 Implementerer toggle-button 👍